src/rules/function-url-scheme-whitelist/README.md

```diff
@@ -0,0 +1,61 @@
+# function-url-scheme-whitelist
+
+Specify a whitelist of allowed url schemes.
+
+```css
+a { background-image: url('http://www.example.com/file.jpg'); }
+/**                        ↑
+ *           This url scheme */
+```
+
+A [url scheme](https://url.spec.whatwg.org/#syntax-url-scheme) consists of alphanumeric, `+`, `-`, and `.` characters. It can appear at the start of a url and is followed by `:`.
+
+This rule ignores url arguments without an existing url scheme.
+
+This rule ignores url arguments that are variables (`$sass`, `@less`, `--custom-property`).
```

Comment:
Am I right in thinking this also ignores interpolation? We should note that.

We should also add a test or two covering these situations.